uint16 stage_remain_time
uint8 game_process
#/* current race stage
#   0 not start
#   1 preparation stage
#   2 self-check stage
#   3 5 seconds count down
#   4 fighting stage
#   5 result computing stage */
uint16 remain_hp
uint16 max_hp
